go-apps/meep-app-enablement/server/service-mgmt/service-mgmt.go

```diff
@@ -147,6 +147,9 @@ func Run() (err error) {
 
 // Stop - Stop Service Mgmt
 func Stop() (err error) {
+	// Flush all app-enablement instance data
```

Comment:
Is there a risk that _basekey_ is not initialized here and we flush the entire redis DB?
We should add a running flag to make sure we are running before stopping the microservice.

Reply:
baseKey is set in the init, so i believe it should have a value that is not "".... but i added a check just in case...